### PR TITLE
fixes prosel. working as nonrat

### LINKED ISSUE #18163: Non-servants can use the clockwork proselytizer
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)
@Xhuis

### PR DESCRIPTION
**Pull request by Iamgoofball**
Fixes #18163

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_objects.dm [hunk 1]
```diff
@@ -194,7 +194,7 @@
 	user << "<i>Inath-Neq: </i>[!clockwork_generals_invoked["inath-neq"] ? "<font color='green'><b>Ready</b></font>" : "<span class='boldannounce'>Recovering</span>"]"
 
 /obj/item/clockwork/slab/proc/show_guide(mob/living/user)
-	var/text = "<font color=#BE8700 size=3><b><center>Chetr nyy hageh�guf naq ubabe Ratvar.</center></b></font><br><br>\
+	var/text = "<font color=#BE8700 size=3><b><center>Chetr nyy hagehguf naq ubabe Ratvar.</center></b></font><br><br>\
```
Comment (Razharas) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: The fuck did you change here?

Comment (Thunder12345) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: Who knows, the exact same line shows as changed for no apparent reason in the other PR

Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: github did it idfk

**Comment by Xhuis** [comment]
When there's an active PR specifically for adding to and improving clockcult, don't make cheap low-effort fix PRs to get credit for feature ones. I'm fixing this in #18085.

**Comment by KorPhaeron** [comment]
stolen code

**Comment by Iamgoofball** [comment]
What the fuck, I coded this one myself

**Comment by Iamgoofball** [comment]
@Xhuis fuck off I didnt know you fixed this until after the PR was opened, you didnt link to the issue report

**Comment by Xhuis** [comment]
That's probably because when I made the PR that fixed the issue, it hadn't been made on the tracker yet. It should also be common knowledge that I've pledged myself to work on clockcult patches until I was happy with the mode.

**Pull request closed without merging**